__tests__/app.test.js

```diff
@@ -163,10 +150,60 @@ describe('app', () => {
               })  
             })
 
-        })
+            })
+            
+            describe('GET/api/articles/:article_id/comments', () => {
+                it('status 200 - successful request should return the correct comment when passed an article_id', () => {
+                    return request(app)
+                        .get('/api/articles/1/comments')
+                        .expect(200)
+                        .then(({ body }) => {
+                            console.log(body)
+                            expect(body.comments).toBeInstanceOf(Array)
+                            expect(body.comments).toHaveLength(11)
+                            body.comments.forEach((comment) => {
+                                expect(comment).toMatchObject({
+                                    comment_id: expect.any(Number),
+                                    votes: expect.any(Number),
+                                    created_at: expect.any(String),
+                                    author: expect.any(String),
+                                    body: expect.any(String),
+                                    article_id: expect.any(Number)
+
 
-       
+                                })
+
+                            })
+                        
 
+                    })
+                });
+
+                it('status 404 - for a valid but non existent article_id', () => {
+                    return request(app)
+                        .get('/api/articles/999/comments')
+                        .expect(404)
+                        .then(({body}) => {
+                        expect(body.msg).toBe('Not Found')
+
+                    })
+                    
+                });
+
+                it('status 400 - for a bad request', () => {
+                    return request(app)
+                        .get('/api/articles/NaN/comments')
+                        .expect(400)
+                        .then(({ body }) => {
+                        
+                            expect(body.msg).toBe('bad request')
+                    })
+                    
+                });
+
+            })
+})
+      })
```

Comment:
Testing looks great here but I would like to see one additional 200 test. What happens if we request comments for an article which exists but has no comments. 

In this case we would expect the api to return an empty array ie,
`{ comments: []}`

As things stand in the model, we would get a 404 error for this test.